### DISCUSSION TITLE
Status of V-2

### DISCUSSION THREAD

**Discussion by IAmSequestered** [smart-on-fhir/smart-launcher-v2#17] [General]
Just a query on the progress status of V-2. Is there a timeline? What's the scope of the project? Do you need/want assistance with any parts of the project?

**Comment by vlad-ignatov** [comment]
It is currently being developed as a new version of https://launch.smarthealthit.org/, adding support for SMART V2 features. Should be released soon (between 1-2 weeks and 1-2 months). Currently deployed to https://smart-launcher-staging.herokuapp.com/.

Assistance is always welcome. Maybe not right now, but once this is officially released, https://github.com/smart-on-fhir/smart-dev-sandbox/ will have to be updated to use it and we could use some help with that.
